src/main/java/com/qaprosoft/carina/demo/mobilepractice/gui/pages/ios/HomePage.java

```diff
@@ -0,0 +1,32 @@
+package com.qaprosoft.carina.demo.mobilepractice.gui.pages.ios;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.foundation.webdriver.locator.ExtendedFindBy;
+import com.qaprosoft.carina.demo.mobilepractice.gui.pages.common.HomePageBase;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = HomePageBase.class)
+public class HomePage extends HomePageBase {
+
+    @ExtendedFindBy(accessibilityId = "ContactsListView")
```

Comment:
this locator finds the whole screen, but you have declared it as List of elements. Find proper locator for contact elements
<img width="824" alt="image" src="https://user-images.githubusercontent.com/103670717/168776358-02913d2f-5518-429a-b992-d274341c6e79.png">

Reply:
I think that I found other way